### PR TITLE
Fix typos and grammar

### PR DESCRIPTION
**Pull request by NirantK**
1. Fixed spelling of "protocol"
2. Rephrased the sentence to make it clear that the protocol is similar to what Whisper used for evaluation

### CODE REVIEW

**Comment by NirantK** [comment]
I've signed the CLA earlier — is there a lag in that propagating here?

File: docs/m4t/on_device_README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Also running the exported model doesn't need python runtime. For example, you co
 
 ## Metrics
 ### S2TT BLEU / S2ST ASR-BLEU on FLEURS
-For ASR-BLEU, we follow the same protocal as Large/Medium models: Use Whisper-large-v2 for eng-X and Whisper-medium for X-eng when evaluating ASR BLEU.
+For ASR-BLEU, we follow the same protocol as Whisper Large/Medium models: We used Whisper-large-v2 for Eng-X and Whisper-medium for X-Eng when evaluating ASR BLEU.
```
Comment (kauterry) [inline docs/m4t/on_device_README.md:37]: It's not `Whisper` Large/Medium models but `SeamlessM4T ` Large/Medium models, please make this change.

**Review by kauterry — CHANGES_REQUESTED** [review]

File: docs/m4t/on_device_README.md (hunk 1 above)
Comment (NirantK) [reply docs/m4t/on_device_README.md]: Done.

Comment (kauterry) [reply docs/m4t/on_device_README.md]: It’s SeamlessM4T Large/Medium models

Comment (NirantK) [reply docs/m4t/on_device_README.md]: Oops, done now with [6431e08](https://github.com/facebookresearch/seamless_communication/pull/14/commits/6431e08962a04ed89e861e9a3a57f0224bbda2e9)

**Review by kauterry — APPROVED** [review]

**Pull request merged**